### ISSUE TITLE
process_stats.py erroring for some loans

### ISSUE THREAD

**Issue by cdrini** [internetarchive/openlibrary#4542]
### Evidence / Screenshot (if possible)
```
2021-02-04 19:47:02 [420] [openlibrary.solrwriter] [INFO] updating loans/loan-1944642791
2021-02-04 19:47:02 [420] [openlibrary.solrwriter] [ERROR] Error in normalizing 'loans/loan-1944635007'
Traceback (most recent call last):
File "/openlibrary/scripts/openlibrary-server", line 133, in
main()
File "/openlibrary/scripts/openlibrary-server", line 130, in main
start_server()
File "/openlibrary/scripts/openlibrary-server", line 46, in start_server
infogami.run(args)
File "/openlibrary/infogami/init.py", line 151, in run
run_action(args[0], args[1:])
File "/openlibrary/infogami/init.py", line 138, in run_action
a(*args)
File "/openlibrary/openlibrary/actions.py", line 11, in runmain
mod.main(*args)
File "/openlibrary/openlibrary/solr/process_stats.py", line 219, in main
add_events_to_solr(events)
File "/openlibrary/openlibrary/solr/process_stats.py", line 210, in add_events_to_solr
update_solr(solrdocs)
File "/openlibrary/openlibrary/solr/process_stats.py", line 261, in update_solr
solr.update(doc)
File "/openlibrary/openlibrary/solr/solrwriter.py", line 57, in update
self.flush()
File "/openlibrary/openlibrary/solr/solrwriter.py", line 64, in flush
node = dict2element(doc)
File "/openlibrary/openlibrary/solr/solrwriter.py", line 108, in dict2element
add_field(doc, k, v)
File "/openlibrary/openlibrary/solr/solrwriter.py", line 98, in add_field
value = value.decode('utf-8')
AttributeError: 'str' object has no attribute 'decode'
```

### Steps to Reproduce

1. `python $SCRIPTS/openlibrary-server /olsystem/etc/openlibrary.yml runmain openlibrary.solr.process_stats --load-from-db`

* Actual: Error
* Expected: No error

### Details
NA

### Stakeholders
@cclauss

**Comment by dherbst** [comment]
The issue is the `.decode` on py3 str does not exist.

**Comment by cclauss** [comment]
Closed by @dherbst in #4545
